### PR TITLE
Update the research file

### PR DESCRIPTION
**Pull request by anranjiao**

### CODE REVIEW

File: docs/user/research.rst [hunk 1]
```diff
@@ -65,16 +65,17 @@ PINN
...
-#. Y. Wang, X. Han, C. Chang, D. Zha, U. Braga-Neto, & X. Hu. `Auto-PINN: Understanding and optimizing physics-informed neural architecture <https://arxiv.org/abs/2205.13748>`_. *arXiv preprint arXiv:2205.13748*, 2022.
+#. A. Serebrennikova, R. Teubler, L. Hoffellner, E. Leitner, U. Hirn, & K. Zojer. `Transport of organic volatiles through paper: Physics-informed neural networks for solving inverse and forward problems <https://link.springer.com/article/10.1007/s11242-022-01864-7>`_. *Transport in Porous Media*, 1-24, 2022.
+#. C. Garcia-Cervera, M. Kessler, & F. Periago. `Control of partial differential equations via physics-informed neural networks <https://link.springer.com/article/10.1007/s10957-022-02100-4>`_. *Journal of Optimization Theory and Applications*, 1-24, 2022.
+#. M. Takamoto, T. Praditia, R. Leiteritz, D. MacKinlay, F. Alesiani, D. Pflüger & M. Niepert. `PDEBENCH: An Extensive Benchmark for Scientific Machine Learning <https://arxiv.org/abs/2210.07182>`_. *arXiv preprint arXiv:2210.07182*, 2022.
```
Comment (lululxvi) [inline docs/user/research.rst:70]: D. Pflüger, & M. Niepert

File: docs/user/research.rst [hunk 2]
```diff
@@ -65,16 +65,17 @@ PINN
...
 #. X. Wang, J. Li, & J. Li. `A deep learning based numerical PDE method for option pricing <https://link.springer.com/article/10.1007/s10614-022-10279-x>`_. *Computational Economics*, 1-16, 2022.
-#. A. Serebrennikova, R. Teubler, L. Hoffellner, E. Leitner, U. Hirn, & K. Zojer. `Transport of organic volatiles through paper: Physics-informed neural networks for solving inverse and forward problems <https://www.researchgate.net/profile/Alexandra-Serebrennikova/publication/360717115_Transport_of_organic_volatiles_through_paper_physics-informed_neural_networks_for_solving_inverse_and_forward_problems/links/6286753e8ecbaa07fcc19c64/Transport-of-organic-volatiles-through-paper-physics-informed-neural-networks-for-solving-inverse-and-forward-problems.pdf>`_. 2022.
+#. Y. Wang, X. Han, C. Chang, D. Zha, U. Braga-Neto, & X. Hu. `Auto-PINN: Understanding and optimizing physics-informed neural architecture <https://arxiv.org/abs/2205.13748>`_. *arXiv preprint arXiv:2205.13748*, 2022.
```
Comment (lululxvi) [inline docs/user/research.rst:72]: Why this paper and the paper above are reordered?

Comment (anranjiao) [reply docs/user/research.rst:72]: I inserted the paper based on the date. Should I keep the previous order?

Comment (lululxvi) [reply docs/user/research.rst:72]: OK. Then it is good.

File: docs/user/research.rst [hunk 3]
```diff
@@ -33,6 +33,7 @@ DeepXDE has been used in
   `University of Johannesburg <https://www.uj.ac.za>`_,
   `University of Surrey <https://www.surrey.ac.uk>`_,
   `University of Los Andes <https://uniandes.edu.co/en>`_,
+  `University of Stuttgart <https://www.uni-stuttgart.de/en/>`_,
```
Comment (lululxvi) [inline docs/user/research.rst:36]: Line 6: > 40 universities

**Pull request merged**